lab_5_scrapper/scrapper.py

```diff
@@ -39,6 +68,7 @@ def get_seed_urls(self) -> list[str]:
         Returns:
             list[str]: Seed urls
         """
+        return self.config.seed_urls
```

Comment:
some conceptual question. why do you think we create these getters that only return attributes of the aggregated object?